src/models/user_account.ts

```diff
@@ -25,8 +32,19 @@ export default class UserAccount extends Model<
 > {
   public id!: number
 
+  @Unique
   @Column
-  public auth0Id!: string
+  public username!: string
```

Comment:
**question (non-blocking):** why require a username instead of an email?

Reply:
Yes, good point. I started this switch without emails, but I realize we need them anyway (for password reset), so I will remove the username field and turn it into an email field.